pkg/pubsub/operations/subscription.go

```diff
@@ -56,7 +63,23 @@ func NewSubscriptionOps(arg SubArgs) *batchv1.Job {
 	}, {
 		Name:  "PUBSUB_SUBSCRIPTION_ID",
 		Value: arg.SubscriptionID,
-	}}...)
+	}}
+
+	switch arg.Action {
+	case ActionCreate:
+		env = append(env, []corev1.EnvVar{{
+			Name:  "PUBSUB_SUBSCRIPTION_CONFIG_ACK_DEAD",
+			Value: arg.AckDeadline.String(),
+		}, {
+			Name:  "PUBSUB_SUBSCRIPTION_CONFIG_RET_ACKED",
+			Value: strconv.FormatBool(arg.RetainAckedMessages),
+		}, {
+			Name:  "PUBSUB_SUBSCRIPTION_CONFIG_RET_DUR",
+			Value: arg.RetentionDuration.String(),
+		}}...)
```

Comment:
It'd be nice if `envconfig` had a method for creating this from the structured type :-/

Reply:
I made an issue to track this idea.